scripts/parse-build.webpack.config.js

```diff
@@ -0,0 +1,24 @@
+var path = require('path');
+
+module.exports = {
+  entry: path.join(__dirname, '..', 'src/server/builds/parse.js'),
+  output: {
+      path: path.join(__dirname, '..', 'dist'),
+      filename: 'algoliasearch.parse.js',
+      libraryTarget: 'commonjs2'
+  },
+  target: 'node',
+  resolve: {
+    alias: {
+      debug: 'debug/browser.js',
```

Comment:
this is where we fallback to browser implems for some requires, so that we get the simple code instead of full nodejs with core requires that will fail in parse.

Parse is like the IE6 of server side JavaScript, easier to get things done this way